lib/active_admin/async_exporter/reports/worker.rb

```diff
@@ -20,6 +20,28 @@ def perform(options = {})
 
       private
 
+      def proc_from_string(code_string)
+        proc_string = code_from_string(code_string)
+
+        Proc.class_eval("proc #{proc_string}")
+      end
+
+      def code_from_string(code_string)
```

Comment:
Wouldn't it be easier to use a Regexp? Something like `/^(do|{)\s+\|.*\|(.*)(end|})$/` Just asking...

Reply:
Will give it a try

Reply:
Zup! Have you tried what Franco proposed? What's your final decision here? 😸

Reply:
Hi! Haven't tried yet, will make sure I give it a try today though!

Reply:
Done!